app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self, name, health=100, hidden=False):
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __sub__(self, other):
+        other.health -= 50
+        if other.health <= 0:
+            Animal.alive.remove(other)
+
+    def __repr__(self):
+        name = f"Name: {self.name}"
+        health = f"Health: {self.health}"
+        hidden = f"Hidden: {self.hidden}"
+        f1 = "{"
+        f2 = "}"
+        return f"{f1}{name}, {health}, {hidden}{f2}"
```

Comment:
write this, without variables

Reply:
It is because of "A single '{'  or '}' is not allowed inside f-strings"

Reply:
![image](https://user-images.githubusercontent.com/107995555/178499900-f9a85963-7750-4a26-8966-cafe81f87aae.png)
![image](https://user-images.githubusercontent.com/107995555/178499999-8de69596-0a71-4b4b-8bfd-bec403194420.png)

Reply:
![Uploading image.png…]()

Reply:
<img width="464" alt="image" src="https://user-images.githubusercontent.com/72562232/178500376-c14d4c0c-05cc-4421-bb2d-2eb7b7d8409a.png">